Added pages for "Contact" and "About" (#128)
mumble-voip/mumble-www: hugo/content/contact.md

---
title: Contact

> 

Reviewer: Remove trailing empty line in front matter

```suggestion
```

---

You can contact us in several ways.

## Community Channels

* [Issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
* Chat:
    * IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)

>     * Matrix: Join `#freenode_#mumble:matrix.org`; for example via [Riot.im (Web App)](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)

Reviewer 2: ```suggestion
    * Matrix: Join `#freenode_#mumble:matrix.org`; for example via [Element (Web App)](https://app.element.io/#/room/#mumble:matrix.org) (shares messages with IRC)
```

---
title: Contact

> toc: true

Reviewer: This seems to enable a table of content.

But I do not see any toc being rendered!?

Author: Well I haven't checked this in the docs, but with other software I use (for some other projects) this tag can have different meanings, for example it means to enable an indexing of the website (so that you have section links to each category on the page).

Reviewer: It’s a parameter that we currently do not use. If we were to enable it here, it could lead to unforeseen consequences once we add toc handling to the layout templates. We don’t know what would happen.

It should be removed to prevent unknown risks while not adding or doing anything.

Author: Ok, just for clarification:
1. Without this option we would still have section links like: https://www.mumble.info/contribute/#report-bugs-and-feature-requests ?
2. Is this a custom parameter?
3. This parameter is used on other pages too, should we remove it everywhere?

Reviewer: Yes, [contribute.md](https://github.com/mumble-voip/mumble-www/blob/master/hugo/content/contribute.md) manually creates a list with links as content, and depends on the headline text which leads to an id attribute to match the link that is declared.

If toc were implemented like this, we would suddenly have two table of content lists.

If a headline changes, the manual list has to be updated too atm.

When I checked our layout did not have toc handling. So yes, it should IMO be removed from other pages too.

Given that the params docs gives an example for `include_toc` as a parameter, I think `toc` as it was used here is a user defined parameter too. There seems to be no other doc on `toc`, which supports that theory.

---

> 
> # Contact
> 

Reviewer: This adds a duplicate headline to the rendered page. Did you miss that on testing?

```suggestion

```

Author: Seems that way. Ok, will use the title tag only instead.

> You can contact us in several ways:

Author: I think that is too long text, but if you want it this way.

## Community Channels

> * Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)

Reviewer: Prefixing/Introing with _Our_ here is inconsistent with the other bullet points.

```suggestion
* [Issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
```

Author: I wanted to clarify with that, that it belongs to mumble.
Maybe I would reformulate it, like:
`[Issue Tracker in Mumble's Github Repo]`

Reviewer 2: I think it should be clear from the context that this is an issue tracker related to Mumble :thinking:

Author: > I think it should be clear from the context that this is an issue tracker related to Mumble :thinking:

That might be correct.

Author: Ok, I will just name it `Issue tracker on GitHub`.

* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)

> * IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
> * Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)

Reviewer: I think this should also be made more consistent. Chat is very unspecific, and IRC is also a chat.

With the other bullet points being mostly full links it seems good to make them here so as well.

```suggestion
* [IRC #mumble on freenode](irc://irc.freenode.org/mumble)
* [Mumble chat room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
```

Author: You are correct with this, but I would change it differently:
```

* Chat:
    * IRC: bla
    * Matrix: bla
```

> * Our [Forums](https://forums.mumble.info/)

Reviewer: ```suggestion
* [Forums](https://forums.mumble.info/)
```

Author: 1. Should we use `Forum` instead?
2. Should I add `Official Forum`?

Reviewer: All of these are official resources. I think just Forum or Forums is fine.

## Email

**General contact:**

> contact [at] mumble [dot] info

Author: Optional: Do you have an encryption key?

Reviewer 2: For email? I don't think so

Author: Yes. Ok.

**For security-related topics:**

security [at] mumble [dot] info

> **For License-related topics:**>

Reviewer 2: ```suggestion
**For License-related topics:**
```

Author: Oh, must have missed that.

---
title: Contact
toc: true
---

# Contact

You can contact us in several ways:

## Community Channels

* Our [issue tracker on GitHub](https://github.com/mumble-voip/mumble/issues)
* [Discussions on GitHub](https://github.com/mumble-voip/mumble/discussions)
* IRC: [#mumble on irc.freenode.net](irc://irc.freenode.org/mumble)
* Chat: [Mumble room on Riot/Matrix](https://riot.im/app/#/room/#mumble:matrix.org) (shares messages with IRC)
* Our [Forums](https://forums.mumble.info/)

## Email

**General contact:**

contact [at] mumble [dot] info

**For security-relevant topics:**

security [at] mumble [dot] info

**For legal matters:**

> <!-- add special adress for legal matters? -->

Author: Do you already have a special emailadress for this and if not, do you want to create one?

Reviewer 2: I don't think so. To my knowledge we never needed that and if there really is something to discuss they can simply use the contact mail address...

Author: Ok.
